automation/taskcluster/androidTest/flank-x86-tablet.yml

```diff
@@ -0,0 +1,54 @@
+# gcloud args match the official gcloud cli
```

Comment:
When do you use this file? I didn't see it in the script below :thinking:

Reply:
> When do you use this file? I didn't see it in the script below 🤔

We don't (currently) use this in the CI, but the devices we want to use in Firebase are specified here, when we'd need to run tests on tablet either manually or via CI.